portfolio/src/main/webapp/script.js

```diff
@@ -27,23 +30,19 @@ function addRandomFunFact() {
   funfactContainer.innerText = funfact;
 }
 
-// TODO: Use following functions in step 5 to display comment
-
+// TODO: Use following functions in step 5 to display comment.
```

Comment:
Remove this comment now that you are displaying comments

Reply:
Done